src/components/CollectionsAndPalettes/Collections/Collection/Collection.tsx

```diff
@@ -31,6 +31,8 @@ class Collection extends React.Component<Props, State> {
   static contextType = AppContext;
   context!: AppContextState;
   modalTarget = null;
+  hotspots: any;
```

Comment:
Please try to use types wherever possible. It makes a bit easier to figure out what would be stored here when looking over the code.